### ISSUE TITLE
ignore individual files

### ISSUE THREAD

**Issue by twhiteaker** [netlify-labs/netlify-plugin-a11y#25]
**Is your feature request related to a problem? Please describe.**
I include a googlexxxxxx.html file to tell Google that the site belongs to me. It fails accessibility, but that isn't an issue to end users since they never see that page.  However, it causes the plugin to fail, and I don't want to switch to warn.

**Describe the solution you'd like**
Add optional input called ignoreFiles with list of files to ignore

```
ignoreFiles = ['/failing-file.html']  # explicitly ignore these files
```

**Describe alternatives you've considered**
I thought of editing the google file to make it accessible, but Google doesn't want you to touch the file I think.

**Additional context**
For more info on the Google file, see the **HTML file upload** section on the [Verify your site ownership page](https://support.google.com/webmasters/answer/9008080?hl=en).
